shared/src/SystemTextJsonWritablePropertyResponse.cs

```diff
@@ -19,13 +19,13 @@ public sealed class SystemTextJsonWritablePropertyResponse : IWritablePropertyRe
         /// <summary>
         /// Convenience constructor for specifying the properties.
         /// </summary>
-        /// <param name="propertyValue">The unserialized property value.</param>
+        /// <param name="value">The unserialized property value.</param>
         /// <param name="ackCode">The acknowledgment code, usually an HTTP Status Code e.g. 200, 400.</param>
         /// <param name="ackVersion">The acknowledgment version, as supplied in the property update request.</param>
         /// <param name="ackDescription">The acknowledgment description, an optional, human-readable message about the result of the property update.</param>
-        public SystemTextJsonWritablePropertyResponse(object propertyValue, int ackCode, long ackVersion, string ackDescription = default)
+        public SystemTextJsonWritablePropertyResponse(object value, int ackCode, long ackVersion, string ackDescription = default)
```

Comment:
In order for us to be able to deserialize a Writable property the constructor parameters have to match the property names.